persistent/cPersistence.c

```diff
@@ -290,7 +290,7 @@ changed(cPersistentObject *self)
         }
         Py_INCREF(self);
         PyTuple_SET_ITEM(arg, 0, (PyObject *)self);
-        result = PyEval_CallObject(meth, arg);
+        result = PyObject_CallObject(meth, arg);
```

Comment:
`PyEval_CallObject` is deprecated in 3.9. `PyObject_CallObject` is a drop-in replacement AFAICS. (An earlier iteration of the change note mentioned something about deprecated functions but I dropped that.)